pkg/os/windows/network/vswitch_windows.go

```diff
@@ -0,0 +1,22 @@
+package network
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/code-ready/crc/pkg/os/windows/powershell"
+)
+
+const hypervDefaultVirtualSwitchId = "c08cb7b8-9b3c-408e-8e30-5e16a3aeb444"
+
+func GetDefaultSwitchName() (bool, string) {
```

Comment:
This returns a boolean which is never used?

Reply:
not in this code path, but for existence in preflight check this has to be used. Pending refactor.